USE_SYSTEM_SUITESPARSE=1 sets wrong include path for suitesparse on FreeBSD

In deps/Makefile the include path for suitesparse is set as `SUITESPARSE_INC = -I /usr/include/suitesparse` (Line 1555).
Which fails in FreeBSD as packages are installed into /usr/local (hence includes in /usr/local/includes) per default and possibly somewhere else. Also on Linux e.g. if you manually install suitesparse from tarball.

For completeness, the error message:
    root@101amd64-system:/wrkdirs/julia # gmake CC=gcc46 CXX=g++46 FC=gfortran46
    SuiteSparse_wrapper.c:2:21: fatal error: cholmod.h: No such file or directory
compilation terminated.
    Makefile:1564: recipe for target
'/wrkdirs/julia/usr/lib/libsuitesparse_wrapper.so' failed
    gmake[1]: **\* [/wrkdirs/julia/usr/lib/libsuitesparse_wrapper.so] Error 1
    Makefile:49: recipe for target 'julia-deps' failed
    gmake: **\* [julia-deps] Error 2

--- a/Make.inc
+++ b/Make.inc
@@ -501,6 +501,12 @@ else
 build_shlibdir := $(build_libdir)
 endif
 
+ifeq ($(OS), FreeBSD)
+LOCALBASE ?= /usr/local
+else
+LOCALBASE ?= /usr
+endif
+
 ifeq (exists, $(shell [ -e $(JULIAHOME)/Make.user ] && echo exists ))
 include $(JULIAHOME)/Make.user
 endif
@@ -837,8 +843,8 @@ LIBMNAME := libopenlibm
 endif
 
 ifeq ($(USE_SYSTEM_LIBUV), 1)
-  LIBUV := /usr/lib/libuv-julia.a
-  LIBUV_INC := /usr/include
+  LIBUV := $(LOCALBASE)/lib/libuv-julia.a
+  LIBUV_INC := $(LOCALBASE)/include
 else
   LIBUV := $(build_libdir)/libuv.a
   LIBUV_INC := $(build_includedir)
@@ -846,7 +852,7 @@ endif
 
 ifeq ($(USE_SYSTEM_UTF8PROC), 1)
   LIBUTF8PROC := -lutf8proc
-  UTF8PROC_INC := /usr/include
+  UTF8PROC_INC := $(LOCALBASE)/include
 else
   LIBUTF8PROC := $(build_libdir)/libutf8proc.a
   UTF8PROC_INC := $(build_includedir)

--- a/deps/suitesparse.mk
+++ b/deps/suitesparse.mk
@@ -101,7 +101,7 @@ install-suitesparse: $(build_prefix)/manifest/suitesparse install-suitesparse-wr
 # SUITESPARSE WRAPPER
 
 ifeq ($(USE_SYSTEM_SUITESPARSE), 1)
-SUITESPARSE_INC := -I /usr/include/suitesparse
+SUITESPARSE_INC := -I $(LOCALBASE)/include/suitesparse
 SUITESPARSE_LIB := -lumfpack -lcholmod -lamd -lcamd -lcolamd -lspqr
 else
 SUITESPARSE_INC := -I $(BUILDDIR)/SuiteSparse-$(SUITESPARSE_VER)/CHOLMOD/Include -I $(BUILDDIR)/SuiteSparse-$(SUITESPARSE_VER)/SuiteSparse_config -I $(BUILDDIR)/SuiteSparse-$(SUITESPARSE_VER)/SPQR/Include

--- a/src/Makefile
+++ b/src/Makefile
@@ -35,6 +35,10 @@ else
 FLAGS += -DJL_BUILD_UNAME='"$(OS)"'
 endif
 
+ifeq ($(OS),FreeBSD)
+FLAGS += -I$(LOCALBASE)/include
+endif
+
 SRCS := \
 	jltypes gf typemap ast builtins module interpreter symbol \
 	dlload sys init task array dump toplevel jl_uv datatype \

